core/src/identity/user/user.service.ts

```diff
@@ -81,6 +81,17 @@ export class UserService {
         });
     }
 
+    async getUserByDiscordId(discordId: string): Promise<User> {
```

Comment:
As mentioned before, similar to something already implemented.

Reply:
[75d4d80](https://github.com/SprocketBot/sprocket/pull/155/commits/75d4d806f2c05d1361b2d1636188703a5ca83a20)